chpl/chpl-service/src/main/java/gov/healthit/chpl/auth/jwt/JWTConsumer.java

```diff
@@ -7,7 +7,7 @@
 public interface JWTConsumer {
 
     /**
-     * Returns map of JWT claims, or returns null if token is invalid 
+     * Returns map of JWT claims, or returns null if token is invalid
      */
-    public Map<String, Object> consume(String jwt) throws InvalidJwtException;
+    Map<String, Object> consume(String jwt) throws InvalidJwtException;
```

Comment:
I feel like this interface and the JWTAuthor interfaces are instances in which we can get rid of the interfaces (like we have been doing with the DAO and DAOImpl, Manager and ManagerImpl, etc. code). If there is no other class that implements these interfaces I *think* we don't need them; the interfaces could be deleted and the implementing class could be renamed to JWTConsumer